### PR TITLE
Feature/installation updates and help

### LINKED ISSUE #15: Corrections to cmfive documentation required
There are a few updates required to the cmfive documentation to make it easier to run the cmfive tutorial.

Eg Adding the models folder to the example folder after the example folder has been created.  The models folder needs to be created to run the demo at this stage.  Changing the wording of the documentation around this change accordingly.

### PR DESCRIPTION
**Pull request by Dane-2pi**
Starting the PR for Alice' changes

### CODE REVIEW

**Comment by Dane-2pi** [comment]
The conflict is going to need some work, Adam's one is more up to date, but Alice's is better written. 
And neither takes codespaces into account, or any other changes over the last 12 months

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, feature/spellcheck ]
```
Comment (Dane-2pi) [inline .github/workflows/main.yml]: drop this again, but it should be on develop

**Comment by Dane-2pi** [comment]
The CI tools are back, 
No deletions, all additions. I've been through every file to look for obvious errors, but not with a base install beside me.

**Review by 2piJareem — APPROVED** [review]
Looks good Dane.  Given that it is documentation I'm happy to approve.

**Review by adam-buckley — APPROVED** [review]

**Pull request merged**